Tests/UITests/TestApp/ScheduleView.swift

```diff
@@ -36,7 +36,7 @@ struct ScheduleView: View {
             EventScheduleList(date: date) { event in
                 if event.task.id == TaskIdentifier.socialSupportQuestionnaire {
                     InstructionsTile(event, alignment: alignment) {
-                        event.complete()
+                        try? event.complete()
```

Comment:
Might be good to test this in the UI tests and therefore shown an alert here as well?

Reply:
The `InstructionsTile` uses the `EventActionButton`, which in turn is already evaluating the completion policy, so this is absolutely fine (if you're not allowed to complete the event, you won't end up in the callback anyway)

Reply:
Sounds fine, should we at least add an assertion in [`InstructionsTile.swift`](https://github.com/StanfordSpezi/SpeziScheduler/pull/58#discussion_r1966952101) highlighting that this error should never be thrown to avoid any regressions in the future that implicitly rely on this behavior?